platform/android/src/native_map_view.cpp

```diff
@@ -391,7 +392,7 @@ void NativeMapView::setReachability(jni::JNIEnv&, jni::jboolean reachable) {
 }
 
 void NativeMapView::resetPosition(jni::JNIEnv&) {
-    map->resetPosition();
+    map->jumpTo(mbgl::CameraOptions().withCenter(mbgl::LatLng {}).withZoom(0.0).withBearing(0.0).withPitch(0.0));
```

Comment:
`mbgl::CameraOptions().withCenter(mbgl::LatLng {}).withZoom(0.0).withBearing(0.0).withPitch(0.0)` is quite long. Would it make sense to add a factory function to create such a CameraOptions?

Reply:
I think it makes sense - though I'm not sure if we should add these on e.g. `CameraOptions` or inside `Map` itself - the latter knows about both camera and animation options.

Reply:
Decided not to pursuit a factory function for now - let's revisit this if platform SDK teams think this is necessary.